.travis.yml

```diff
@@ -1,4 +1,12 @@
 language: scala
 scala:
-   - 2.10.4
-   - 2.11.2
+- 2.10.4
```

Comment:
Any reason it is not the same 2.10.5 and 2.11.6 versions as in the build.sbt file?